compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/expressions/QueryExpressionTest.java

```diff
@@ -241,4 +241,9 @@ public void testIncompleteClauses() {
         testFile("query-expr/query_expr_source_65.bal", "query-expr/query_expr_assert_65.json");
         testFile("query-expr/query_expr_source_66.bal", "query-expr/query_expr_assert_66.json");
     }
+
+    @Test
+    public void testIntermediateClauseStartRecovery() {
+        testFile("query-expr/query_expr_source_68.bal", "query-expr/query_expr_assert_68.json");
```

Comment:
`testFile` starts context with `TOP_LEVEL_NODE`. Let's add the codes inside a function

Reply:
oops. thanks!